sklearn/feature_extraction/hashing.py

```diff
@@ -117,6 +117,10 @@ def fit(self, X=None, y=None):
         This method doesn't do anything. It exists purely for compatibility
         with the scikit-learn transformer API.
 
+        Parameters
+        ----------
+        X : array-like
```

Comment:
It's not actually used, but it's not expected to be an array. It can be any iterable that matches the "iterable over iterable over raw features" as for the transform method.